loadtest/user/userentity/user.go

```diff
@@ -22,6 +22,7 @@ type UserEntity struct {
 	wsClient    *model.WebSocketClient
 	closing     chan struct{}
 	closed      chan struct{}
```

Comment:
Any opinion on adding the `ws` prefix also to these two? Might seem a bit vague otherwise.